dbt_subprojects/dex/models/_projects/zeroex/zeroex_native_trades.sql

```diff
@@ -21,7 +21,7 @@ FROM (
     {% for model in zeroex_models %}
     SELECT
       blockchain  as blockchain,
-      '0x API'  as project,
+      '0x-API'  as project,
```

Comment:
@RantumBits do you approve of this project name update? more detail in PR description. we are open to another value other than `-`, if you'd prefer (`0x_API`, `0xAPI`, etc.)

Reply:
0x-API is good